apps/dev-recruiters/src/components/modules/FormPage/signUpForm.tsx

```diff
@@ -83,6 +157,53 @@ export default function SignUpForm({
     setShowConfirmationModal(true);
   };
 
+  const [showUploadModal, setShowUploadModal] = useState(false);
+  const [uploadError, setUploadError] = useState('');
+
+  const [canButVis, setCanButVis] = useState(true);
+
+  const handleUploadOpenModal = () => {
+    setSelectedFiles([]);
+    setFilesUploaded({});
+    setUploadError('');
+    setCanButVis(true);
+    setShowUploadModal(true);
+  };
+
+  const handleUploadCloseModal = () => {
+    setShowUploadModal(false);
+  };
+  const handleOkCloseModal = async () => {
+    setShowUploadModal(false);
+    setIsUploading(true);
+  };
+  const handleCancelCloseModal = () => {
+    setSelectedFiles([]);
+    setFilesUploaded([]);
+    setShowUploadModal(false);
+  };
+
+  const handleRemoveFile = () => {
+    setIsDeleting(true); // Deleting state
+    const newArr = [...selectedFiles];
+    axios
+      .delete(
+        `${process.env.NEXT_PUBLIC_STRAPI_URL}/googledrive/${filesUploaded['id']}`
```

Comment:
We try to keep all the logic to query backend in https://github.com/dev-launchers/dev-launchers-platform/blob/master/packages/utility/agent.ts